tests/braille-specs/spaces_bulk_test_part1.yaml

```diff
@@ -0,0 +1,176 @@
+# Bulk testing international tables two spaces.uti related
+# modifications (\x2060 and \xfeff character replacement)
+# Because the Yaml system enable only 150 table/yaml file, need
+# separate this tests with two parts.
```

Comment:
I suggest we increase `MAXTABLES` to something higher than 154.

Also, I would add this whole test to spaces.yaml.